content/tools/polyfill.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Polyfills
+sub: Cascade Layers, CSS Anchor Positioning, CSS Toggles, Popover
```

Comment:
I think the CSS Toggles proposal has been abandoned, so there's not much point marketing the polyfill as a viable tool for people to use. At most I think we could note it at the end of the post as an abandoned exploration - but shouldn't feature it here.